SonarQube.Common/ProcessRunnerArguments.cs

```diff
@@ -26,12 +26,16 @@ public class ProcessRunnerArguments
         /// Strings that are used to indicate arguments that contain
         /// sensitive data that should not be logged
         /// </summary>
-        public static readonly string[] SensitivePropertyKeys = new string[] {
-            SonarProperties.SonarPassword,
-            SonarProperties.SonarUserName,
-            SonarProperties.DbPassword,
-            SonarProperties.DbUserName
-        };
+        public static readonly string[] SensitivePropertyKeys = new string[] { };
+        // SONARMSBRU-136/SONARMSBRU-137: back-compat with v1.0 - allow sensitive data to be stored in the generated project properties file and config file.
+        // SONARMSBRU-136: TODO - uncomment the following code:
+        //public static readonly string[] SensitivePropertyKeys = new string[] {
+        //    SonarProperties.SonarPassword,
+        //    SonarProperties.SonarUserName,
+        //    SonarProperties.DbPassword,
+        //    SonarProperties.DbUserName
+        //};
+
```

Comment:
Note that there are other config/settings changes between v1.0.1 and v1.0 e.g. in v1.0.1 settings are not copied from the /s: settings file into the config. However, we are only interested in compatibility between the v1.0 bootstrapper and the v1.0.1 plug-in bits, and the bootstrapper never reads the config file.